config/database.yml

```
@@ -0,0 +1,42 @@
+# MySQL.  Versions 4.1 and 5.0 are recommended.
+# 
+# Install the MYSQL driver
+#   gem install mysql2
+#
+# Ensure the MySQL gem is defined in your Gemfile
+#   gem 'mysql2'
+#
+# And be sure to use new-style password hashing:
+#   http://dev.mysql.com/doc/refman/5.0/en/old-client.html
+development:
+  adapter: mysql2
+  encoding: utf8
+  reconnect: false
+  database: mysms_development
+  pool: 5
+  username: root
+  password: G0n3ad5o7
```

Usually we would set the password for development/test databases to something simple like "example" or "password" just so it's clear that it isn't a production password

---

Also, you could allow these to be specified via enviornment variables which would help with the docker setup, and fallback to some sane defaults

 ```suggestion
  password: <%= ENV['DB_PASSWORD'] || 'example' %>
```